### DISCUSSION TITLE
cannot import name 'MigrateCommand' from 'flask_migrate'

### DISCUSSION THREAD

**Discussion by vanloctech** [miguelgrinberg/Flask-Migrate#418] [Q&A] [answered]
File "manage.py", line 4, in <module>
    from flask_migrate import Migrate, MigrateCommand
ImportError: cannot import name 'MigrateCommand' from 'flask_migrate'

**Comment by miguelgrinberg** [comment, marked as answer]
Flask-Script is not supported anymore. Switch to the Flask CLI or downgrade Flask-Migrate to version 2.7.0, which is the last to support Flask-Script.

**Reply by simpleth** [reply]
how to use the Flask CLI. I am a novice coder

**Comment by MichaelNjoroge254** [comment]
Traceback (most recent call last):
  File "manage.py", line 4, in <module>
    from  flask_migrate import Migrate, MigrateCommand
ImportError: cannot import name 'MigrateCommand' from 'flask_migrate' 


Use pip install flask-migrate==2.1.1 to solve

**Comment by Marcos8060** [comment]
it should work fine after installing the 2.6.0
pip install Flask-Migrate==2.6.0

**Comment by RichieOrito** [comment]
you need to downgrade flask-migrate to a lower version between 2.1.1 to 2.7.0 to solve the Import error. pip install flask-migrate==<a lower version>
